pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1095,7 +1095,7 @@ func (dm *DockerManager) pluginDisablesDockerNetworking() bool {
 
 // newDockerVersion returns a semantically versioned docker version value
```

Comment:
It's used just below on line number 1128 other than that yeah it doesn't seem to be used.